dask/array/percentile.py

```diff
@@ -10,30 +9,6 @@
 from .core import Array
 
 
-@wraps(np.percentile)
```

Comment:
I understand this is mostly an organizational change, but why do we need to move it to the `dask.dataframe` since it's a NumPy-based function, and thus more suitable as `dask.array`?